src/components/plots/piePlot.ts

```diff
@@ -67,6 +79,82 @@ export module Plots {
+    public sectorValue(sectorValue: number | _Accessor): Plots.Pie<D>;
+    public sectorValue(sectorValue: D | _Accessor, sectorValueScale: Scale<D, number>): Plots.Pie<D>;
+    public sectorValue(sectorValue?: number | _Accessor | D, sectorValueScale?: Scale<D, number>): any {
+      if (sectorValue == null) {
+        return Pie._constructBinding(this._sectorValue, this._sectorValueScale);
+      }
+      this._sectorValue = d3.functor(sectorValue);
+      Pie._replaceScaleBinding(this._sectorValueScale, sectorValueScale, this._renderCallback);
+      this._sectorValueScale = sectorValueScale;
+      this._render();
+      return this;
+    }
+
+    public innerRadius(): AccessorScaleBinding<D, number>;
+    public innerRadius(innerRadius: number | _Accessor): Plots.Pie<D>;
+    public innerRadius(innerRadius: D | _Accessor, innerRadiusScale: Scale<D, number>): Plots.Pie<D>;
+    public innerRadius(innerRadius?: number | _Accessor | D, innerRadiusScale?: Scale<D, number>): any {
+      if (innerRadius == null) {
+        return Pie._constructBinding(this._innerRadius, this._innerRadiusScale);
+      }
+      this._innerRadius = d3.functor(innerRadius);
+      Pie._replaceScaleBinding(this._innerRadiusScale, innerRadiusScale, this._renderCallback);
+      this._innerRadiusScale = innerRadiusScale;
+      this._render();
+      return this;
+    }
+
+    public outerRadius(): AccessorScaleBinding<D, number>;
+    public outerRadius(outerRadius: number | _Accessor): Plots.Pie<D>;
+    public outerRadius(outerRadius: D | _Accessor, outerRadiusScale: Scale<D, number>): Plots.Pie<D>;
+    public outerRadius(outerRadius?: number | _Accessor | D, outerRadiusScale?: Scale<D, number>): any {
+      if (outerRadius == null) {
+        return Pie._constructBinding(this._outerRadius, this._outerRadiusScale);
+      }
+      this._outerRadius = d3.functor(outerRadius);
+      Pie._replaceScaleBinding(this._outerRadiusScale, outerRadiusScale, this._renderCallback);
+      this._outerRadiusScale = outerRadiusScale;
+      this._render();
+      return this;
+    }
+
+    public scaledInnerRadiusAccessor(): _Accessor {
+      return Pie._scaledValueAccessor(this._innerRadius, this._innerRadiusScale);
+    }
+
+    public scaledOuterRadiusAccessor(): _Accessor {
+      return Pie._scaledValueAccessor(this._outerRadius, this._outerRadiusScale);
+    }
+
+    public scaledSectorValueAccessor(): _Accessor {
+      return Pie._scaledValueAccessor(this._sectorValue, this._sectorValueScale);
+    }
+
+    private static _scaledValueAccessor<V, SD, SR>(value: _Accessor, scale: Scale<SD, SR>): _Accessor {
+      return scale == null ?
+               value :
+               (d: any, i: number, u: any, m: Plots.PlotMetadata) => scale.scale(value(d, i, u, m));
+    }
+
+    private static _constructBinding<V, D, R>(value: V | _Accessor, scale: Scale<D, number>) {
+      return scale == null ? { accessor: value } : { accessor: value, scale: scale };
```

Comment:
Why is this necessary? It's seems like there'd be less misdirection if we just wrote
```Typescript
binding = {
  accessor: accessor,
  scale: scale
};
```
every time we needed it. If `scale` is `null` or `undefined` it still works correctly.